pkg/resource/workspace/hooks.go

```diff
@@ -0,0 +1,219 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package workspace
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws-controllers-k8s/prometheusservice-controller/apis/v1alpha1"
+	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
+	ackcondition "github.com/aws-controllers-k8s/runtime/pkg/condition"
+	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
+	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
+	svcsdk "github.com/aws/aws-sdk-go/service/prometheusservice"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Can you please split the local import paths from the external ones? we try to follow the standards mentioned in https://github.com/aws-controllers-k8s/lambda-controller/blob/main/pkg/resource/function/hooks.go#L117